{"trial_id": "0001", "hyperparameters": {"space": [{"class_name": "Choice", "config": {"name": "units", "default": 25, "conditions": [], "values": [25, 50, 100], "ordered": true}}, {"class_name": "Choice", "config": {"name": "learning_rate", "default": 0.001, "conditions": [], "values": [0.001, 0.0001, 1e-05], "ordered": true}}, {"class_name": "Choice", "config": {"name": "batch_size", "default": 8, "conditions": [], "values": [8, 16, 32], "ordered": true}}], "values": {"units": 25, "learning_rate": 0.001, "batch_size": 16}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/home/mgsilva/miniconda3/envs/motifpredenv/lib/python3.9/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/home/mgsilva/miniconda3/envs/motifpredenv/lib/python3.9/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/tmp/ipykernel_20754/2916941362.py\", line 31, in run_trial\n    print(X_ttrain_indicesrain)\nNameError: name 'X_ttrain_indicesrain' is not defined\n"}